packages/swirl-components/src/components/swirl-modal/swirl-modal.css

```diff
@@ -429,6 +433,12 @@
   }
 }
 
+.modal--remove-secondary-content-padding {
```

Comment:
Idea: Works for your use case. Adding a prop to control the padding, instead of just disabling it would be more flexible.

Reply:
Like swirl-box does for example. 
`@Prop() padding?: SwirlBoxPadding = "0";`

Reply:
Hmm, nice true, I'll add it

Reply:
added the options, works locally